champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/warlock/passives/SoulHarvest.java

```diff
@@ -26,7 +27,7 @@
 
 @Singleton
 @BPvPListener
-public class SoulHarvest extends Skill implements PassiveSkill {
+public class SoulHarvest extends Skill implements PassiveSkill, BuffSkill {
```

Comment:
heal skill, if regeneration qualifies

Reply:
I don't think I did this with other regeneration skills, just left it as a buff skill. Though is a gray area.